S7.Net/Types/Struct.cs

```diff
@@ -75,11 +76,15 @@ public static int GetStructSize(Type structType)
                         break;
                     default:
                         numBytes += GetStructSize(info.FieldType);
+                        if (count != infos.Count()) {
+                            if(numBytes % 2 != 0) 
+							    numBytes++;//word align
```

Comment:
```suggestion
                        if (count != infos.Count()) 
                        {
                            if (numBytes % 2 != 0) 
							    numBytes++;     //word align
```

Reply:
In a block of the PLC, two structures A and B are defined. Structure A occupies 3-byte space, and structure B also occupies 3-byte space. However, in reality, structure B starts from the fourth byte because the PLC is two-byte aligned internally. 

当在PLC一个块中，定义一个结构体A，B，其中A占用3个字节空间，B也占3个字节空间，但实际上B是从第4字节开始的，因为PLC内部是两字节对齐的。
将以上翻译为英文

Reply:
The above code has been verified by the project and the test is normal.

Reply:
My suggestion is about code style (see the spacing and newline).